ui-core/src/styles/inputs/input.css

```diff
@@ -16,6 +16,27 @@
     }
   }
 
+  .input-container {
+    display: flex;
+    align-items: center;
+    position: relative;
+    width: 100%;
+  }
+
+  .input-icons {
+    position: absolute;
+    display: flex;
+    align-items: center;
+    right: 1rem;
```

Comment:
Double check the sketch, the padding right changes if the input is small or not and probably the spacing between icons when they are both displayed.

Reply:
Checked :)

Reply:
Sorry but I don't see any changes. There is still `1rem` for the `right` property. 

It should be (in rem equivalent) : 
- `9px` with `small` prop
![Capture d’écran 2024-08-08 à 11 20 46](https://github.com/user-attachments/assets/7f5e879c-4444-4baa-b71b-fe2909deaf3e)

- `11px` when no small
![Capture d’écran 2024-08-08 à 11 20 57](https://github.com/user-attachments/assets/0d416487-a049-4c1b-bb8a-3f48c349fbfe)

- `8px` between the two icons (no spacing for now)
![Capture d’écran 2024-08-08 à 11 21 03](https://github.com/user-attachments/assets/a921c988-e79e-4d52-9fde-fc6679db06bf)

Reply:
Done